src/client/components/input/drag-and-drop-image.js

```diff
@@ -1,11 +1,26 @@
 const React = require('react');
 const Input = require('react-bootstrap').Input;
 
+/**
+* This class is derived from the React Component base class to render an image
+* which supports drag and drop functionality.
+*/
 class DragAndDropImage extends React.Component {
+	/**
+	* Binds the class methods to their respective data.
+	* @constructor
+	*/
 	constructor() {
 		super();
 		this.handleDragStart = this.handleDragStart.bind(this);
 	}
+	/**
+	* Transfers the data of the achievement badge component properties to the
```

Comment:
This is better, but still not specific enough. Take a look at https://github.com/bookbrainz/bookbrainz-site/blob/master/src/client/components/input/drag-and-drop.js#L28 in particular - that function is where this data ends up. Then come back to this comment and describe where the data goes, and what it's used for and how this happens.